index.html

```diff
@@ -73,7 +74,7 @@
             <button id="run-btn" onclick="onRun()" disabled>Run</button>
           </div>
           <div class="navbar-compile navbar-item">
-            <a href="#" class="run-btn">computeMain (Compute) -> SPIRV</a>
+            <a href="#" class="run-btn">computeMain (Compute) -> WGSL</a>
             <button id="compile-btn" onclick="onCompile()" disabled>
               Compile
             </button>
```

Comment:
I think it will be more intuitive if there was Compile button on the left side and Run button on the right side.
Because the flow is from left to right and I am supposed to click on Compile button first and Run button later, isn't it?

Reply:
no, you just need to click Run button.
https://docs.google.com/document/d/1wu7nKhdijIOhDYm-Oz9uLqq0BFJQc0LDZUp4sk9mjYo/edit?tab=t.0